[2023-01-04T21:54:36.166+0100] {processor.py:153} INFO - Started process (PID=58) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:54:36.167+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T21:54:36.168+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:54:36.167+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:54:36.188+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:54:36.187+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T21:54:36.188+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:54:36.188+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T21:54:36.188+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:54:36.197+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-01-04T21:58:11.707+0100] {processor.py:153} INFO - Started process (PID=99) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:58:11.708+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T21:58:11.709+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:58:11.708+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:58:11.727+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:58:11.726+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T21:58:11.727+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:58:11.727+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T21:58:11.727+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:58:11.735+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-04T21:58:52.010+0100] {processor.py:153} INFO - Started process (PID=143) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:58:52.011+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T21:58:52.012+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:58:52.012+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:58:52.028+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:58:52.027+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T21:58:52.028+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:58:52.028+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T21:58:52.028+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:58:52.036+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T21:59:30.356+0100] {processor.py:153} INFO - Started process (PID=181) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:59:30.357+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T21:59:30.358+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:59:30.358+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:59:30.371+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:59:30.371+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T21:59:30.371+0100] {logging_mixin.py:137} INFO - [2023-01-04T21:59:30.371+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T21:59:30.371+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T21:59:30.378+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-01-04T22:00:05.888+0100] {processor.py:153} INFO - Started process (PID=217) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:00:05.889+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:00:05.890+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:00:05.890+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:00:05.904+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:00:05.904+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:00:05.904+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:00:05.904+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:00:05.904+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:00:05.912+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:00:42.574+0100] {processor.py:153} INFO - Started process (PID=252) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:00:42.575+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:00:42.576+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:00:42.576+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:00:42.589+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:00:42.589+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:00:42.589+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:00:42.589+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:00:42.589+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:00:42.597+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:01:17.080+0100] {processor.py:153} INFO - Started process (PID=288) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:01:17.081+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:01:17.081+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:01:17.081+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:01:17.095+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:01:17.095+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:01:17.095+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:01:17.095+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:01:17.096+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:01:17.103+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:01:55.187+0100] {processor.py:153} INFO - Started process (PID=326) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:01:55.190+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:01:55.191+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:01:55.191+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:01:55.213+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:01:55.213+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:01:55.213+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:01:55.213+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:01:55.214+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:01:55.222+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-01-04T22:02:32.112+0100] {processor.py:153} INFO - Started process (PID=361) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:02:32.113+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:02:32.114+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:02:32.114+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:02:32.134+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:02:32.133+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:02:32.135+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:02:32.135+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:02:32.135+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:02:32.143+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-01-04T22:03:05.963+0100] {processor.py:153} INFO - Started process (PID=393) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:03:05.967+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:03:05.968+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:03:05.968+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:03:05.993+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:03:05.992+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:03:05.993+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:03:05.993+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:03:05.993+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:03:06.002+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2023-01-04T22:03:36.317+0100] {processor.py:153} INFO - Started process (PID=422) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:03:36.319+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:03:36.320+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:03:36.319+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:03:36.338+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:03:36.338+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:03:36.338+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:03:36.338+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:03:36.338+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:03:36.346+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-01-04T22:04:07.616+0100] {processor.py:153} INFO - Started process (PID=453) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:04:07.619+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:04:07.620+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:04:07.619+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:04:07.633+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:04:07.632+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:04:07.633+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:04:07.633+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:04:07.633+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:04:07.641+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:04:47.468+0100] {processor.py:153} INFO - Started process (PID=491) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:04:47.469+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:04:47.470+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:04:47.469+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:04:47.491+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:04:47.491+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:04:47.492+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:04:47.491+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:04:47.492+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:04:47.509+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-01-04T22:05:21.439+0100] {processor.py:153} INFO - Started process (PID=523) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:05:21.440+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:05:21.441+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:05:21.441+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:05:21.455+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:05:21.455+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:05:21.456+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:05:21.455+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:05:21.456+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:05:21.465+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:05:51.867+0100] {processor.py:153} INFO - Started process (PID=553) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:05:51.868+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:05:51.869+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:05:51.869+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:05:51.882+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:05:51.882+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:05:51.882+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:05:51.882+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:05:51.882+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:05:51.890+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:06:27.478+0100] {processor.py:153} INFO - Started process (PID=588) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:06:27.479+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:06:27.479+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:06:27.479+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:06:27.494+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:06:27.493+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:06:27.494+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:06:27.494+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:06:27.494+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:06:27.502+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:07:03.990+0100] {processor.py:153} INFO - Started process (PID=624) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:07:03.991+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:07:03.991+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:07:03.991+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:07:04.005+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:07:04.004+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:07:04.005+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:07:04.005+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:07:04.005+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:07:04.012+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:07:42.048+0100] {processor.py:153} INFO - Started process (PID=661) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:07:42.050+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:07:42.052+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:07:42.051+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:07:42.078+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:07:42.077+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:07:42.078+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:07:42.078+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:07:42.078+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:07:42.088+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-01-04T22:08:18.677+0100] {processor.py:153} INFO - Started process (PID=698) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:08:18.678+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:08:18.678+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:08:18.678+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:08:18.691+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:08:18.691+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:08:18.691+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:08:18.691+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:08:18.692+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:08:18.699+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-01-04T22:08:52.306+0100] {processor.py:153} INFO - Started process (PID=731) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:08:52.307+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:08:52.308+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:08:52.308+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:08:52.325+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:08:52.324+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:08:52.325+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:08:52.325+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:08:52.325+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:08:52.469+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.166 seconds
[2023-01-04T22:09:26.361+0100] {processor.py:153} INFO - Started process (PID=765) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:09:26.362+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:09:26.362+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:09:26.362+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:09:26.377+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:09:26.377+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:09:26.377+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:09:26.377+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:09:26.377+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:09:26.386+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:10:00.295+0100] {processor.py:153} INFO - Started process (PID=798) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:10:00.296+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:10:00.297+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:10:00.297+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:10:00.313+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:10:00.313+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:10:00.313+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:10:00.313+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:10:00.313+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:10:00.321+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-04T22:10:39.197+0100] {processor.py:153} INFO - Started process (PID=836) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:10:39.198+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:10:39.199+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:10:39.199+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:10:39.212+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:10:39.212+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:10:39.212+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:10:39.212+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:10:39.212+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:10:39.220+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:11:13.945+0100] {processor.py:153} INFO - Started process (PID=871) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:11:13.946+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:11:13.946+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:11:13.946+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:11:13.959+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:11:13.959+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:11:13.960+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:11:13.960+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:11:13.960+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:11:13.967+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:11:53.162+0100] {processor.py:153} INFO - Started process (PID=908) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:11:53.163+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:11:53.164+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:11:53.164+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:11:53.182+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:11:53.181+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:11:53.182+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:11:53.182+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:11:53.182+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:11:53.192+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-01-04T22:12:23.770+0100] {processor.py:153} INFO - Started process (PID=938) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:12:23.771+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:12:23.772+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:12:23.772+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:12:23.790+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:12:23.789+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:12:23.790+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:12:23.790+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:12:23.790+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:12:23.798+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-04T22:12:55.170+0100] {processor.py:153} INFO - Started process (PID=968) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:12:55.171+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:12:55.172+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:12:55.171+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:12:55.185+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:12:55.185+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:12:55.185+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:12:55.185+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:12:55.185+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:12:55.193+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:13:30.278+0100] {processor.py:153} INFO - Started process (PID=1004) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:13:30.279+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:13:30.280+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:13:30.279+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:13:30.294+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:13:30.293+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:13:30.294+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:13:30.294+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:13:30.294+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:13:30.303+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-04T22:14:02.185+0100] {processor.py:153} INFO - Started process (PID=1036) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:14:02.186+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:14:02.186+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:14:02.186+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:14:02.200+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:14:02.199+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:14:02.200+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:14:02.200+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:14:02.200+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:14:02.207+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-01-04T22:14:35.249+0100] {processor.py:153} INFO - Started process (PID=1068) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:14:35.250+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:14:35.251+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:14:35.251+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:14:35.270+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:14:35.269+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:14:35.270+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:14:35.270+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:14:35.270+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:14:35.277+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-04T22:15:09.794+0100] {processor.py:153} INFO - Started process (PID=1102) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:15:09.796+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:15:09.797+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:15:09.796+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:15:09.812+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:15:09.812+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:15:09.812+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:15:09.812+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:15:09.813+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:15:09.820+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:15:46.453+0100] {processor.py:153} INFO - Started process (PID=1138) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:15:46.454+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:15:46.454+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:15:46.454+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:15:46.468+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:15:46.467+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:15:46.468+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:15:46.468+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:15:46.468+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:15:46.476+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:16:22.754+0100] {processor.py:153} INFO - Started process (PID=1174) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:16:22.755+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:16:22.756+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:16:22.756+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:16:22.769+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:16:22.768+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:16:22.769+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:16:22.769+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:16:22.769+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:16:22.779+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:17:00.521+0100] {processor.py:153} INFO - Started process (PID=1212) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:17:00.522+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:17:00.523+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:17:00.523+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:17:00.539+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:17:00.539+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:17:00.539+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:17:00.539+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:17:00.540+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:17:00.547+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:17:31.776+0100] {processor.py:153} INFO - Started process (PID=1244) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:17:31.777+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:17:31.778+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:17:31.778+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:17:31.800+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:17:31.800+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:17:31.801+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:17:31.800+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:17:31.801+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:17:31.809+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-01-04T22:18:09.703+0100] {processor.py:153} INFO - Started process (PID=1282) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:18:09.704+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:18:09.704+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:18:09.704+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:18:09.717+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:18:09.717+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:18:09.717+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:18:09.717+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:18:09.717+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:18:09.725+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-01-04T22:18:45.149+0100] {processor.py:153} INFO - Started process (PID=1317) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:18:45.150+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:18:45.150+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:18:45.150+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:18:45.166+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:18:45.166+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:18:45.166+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:18:45.166+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:18:45.166+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:18:45.175+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:19:27.042+0100] {processor.py:153} INFO - Started process (PID=1363) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:19:27.044+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:19:27.045+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:19:27.045+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:19:27.068+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:19:27.068+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:19:27.069+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:19:27.069+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:19:27.069+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:19:27.078+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-01-04T22:19:57.445+0100] {processor.py:153} INFO - Started process (PID=1512) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:19:57.446+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:19:57.447+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:19:57.447+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:19:57.460+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:19:57.460+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:19:57.460+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:19:57.460+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:19:57.460+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:19:57.469+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:20:37.110+0100] {processor.py:153} INFO - Started process (PID=1548) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:20:37.112+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:20:37.113+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:20:37.113+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:20:37.128+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:20:37.128+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:20:37.128+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:20:37.128+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:20:37.128+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:20:37.136+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-04T22:21:14.074+0100] {processor.py:153} INFO - Started process (PID=1584) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:21:14.145+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:21:14.146+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:21:14.146+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:21:14.174+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:21:14.173+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:21:14.174+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:21:14.174+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:21:14.175+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:21:14.190+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.120 seconds
[2023-01-04T22:21:53.420+0100] {processor.py:153} INFO - Started process (PID=1654) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:21:53.421+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:21:53.422+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:21:53.422+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:21:53.436+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:21:53.436+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:21:53.436+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:21:53.436+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:21:53.437+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:21:53.445+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:22:27.571+0100] {processor.py:153} INFO - Started process (PID=1687) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:22:27.572+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:22:27.573+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:22:27.572+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:22:27.588+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:22:27.587+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:22:27.588+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:22:27.588+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:22:27.588+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:22:27.597+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:22:58.912+0100] {processor.py:153} INFO - Started process (PID=1717) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:22:58.913+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:22:58.913+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:22:58.913+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:22:58.927+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:22:58.926+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:22:58.927+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:22:58.927+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:22:58.927+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:22:58.935+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:23:33.418+0100] {processor.py:153} INFO - Started process (PID=1752) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:23:33.419+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:23:33.420+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:23:33.419+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:23:33.433+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:23:33.432+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:23:33.433+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:23:33.433+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:23:33.433+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:23:33.441+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:24:03.823+0100] {processor.py:153} INFO - Started process (PID=1780) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:24:03.824+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:24:03.825+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:24:03.825+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:24:03.838+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:24:03.838+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:24:03.839+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:24:03.838+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:24:03.839+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:24:03.846+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:24:33.986+0100] {processor.py:153} INFO - Started process (PID=1810) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:24:33.986+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:24:33.987+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:24:33.987+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:24:34.000+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:24:34.000+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:24:34.001+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:24:34.001+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:24:34.001+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:24:34.009+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:25:10.503+0100] {processor.py:153} INFO - Started process (PID=1845) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:25:10.504+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:25:10.505+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:25:10.505+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:25:10.520+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:25:10.519+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:25:10.520+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:25:10.520+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:25:10.520+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:25:10.547+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2023-01-04T22:25:48.504+0100] {processor.py:153} INFO - Started process (PID=1881) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:25:48.507+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:25:48.509+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:25:48.508+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:25:48.530+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:25:48.530+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:25:48.531+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:25:48.531+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:25:48.531+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:25:48.539+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-01-04T22:26:25.569+0100] {processor.py:153} INFO - Started process (PID=1915) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:26:25.571+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:26:25.572+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:26:25.572+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:26:25.591+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:26:25.591+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:26:25.592+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:26:25.591+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:26:25.592+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:26:25.601+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-04T22:26:57.169+0100] {processor.py:153} INFO - Started process (PID=1959) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:26:57.170+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:26:57.172+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:26:57.171+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:26:57.189+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:26:57.189+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:26:57.190+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:26:57.190+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:26:57.190+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:26:57.198+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-04T22:27:29.067+0100] {processor.py:153} INFO - Started process (PID=1991) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:27:29.068+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:27:29.068+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:27:29.068+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:27:29.084+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:27:29.084+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:27:29.084+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:27:29.084+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:27:29.085+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:27:29.092+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-04T22:28:06.239+0100] {processor.py:153} INFO - Started process (PID=2031) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:28:06.240+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:28:06.241+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:28:06.241+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:28:06.256+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:28:06.256+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:28:06.257+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:28:06.257+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:28:06.257+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:28:06.265+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:28:42.242+0100] {processor.py:153} INFO - Started process (PID=2069) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:28:42.243+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:28:42.244+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:28:42.244+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:28:42.264+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:28:42.264+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:28:42.265+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:28:42.265+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:28:42.265+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:28:42.273+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-04T22:29:12.534+0100] {processor.py:153} INFO - Started process (PID=2099) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:29:12.535+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:29:12.536+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:29:12.536+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:29:12.549+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:29:12.549+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:29:12.550+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:29:12.550+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:29:12.550+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:29:12.557+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:29:50.442+0100] {processor.py:153} INFO - Started process (PID=2143) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:29:50.443+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:29:50.443+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:29:50.443+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:29:50.459+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:29:50.458+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:29:50.459+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:29:50.459+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:29:50.459+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:29:50.469+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-01-04T22:30:20.787+0100] {processor.py:153} INFO - Started process (PID=2173) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:30:20.788+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:30:20.789+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:30:20.788+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:30:20.805+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:30:20.804+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:30:20.805+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:30:20.805+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:30:20.805+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:30:20.813+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-04T22:31:00.146+0100] {processor.py:153} INFO - Started process (PID=2217) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:31:00.147+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:31:00.148+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:31:00.148+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:31:00.162+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:31:00.162+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:31:00.163+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:31:00.163+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:31:00.163+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:31:00.172+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-04T22:31:30.686+0100] {processor.py:153} INFO - Started process (PID=2247) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:31:30.687+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:31:30.688+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:31:30.687+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:31:30.705+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:31:30.704+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:31:30.705+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:31:30.705+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:31:30.705+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:31:30.714+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-04T22:32:02.755+0100] {processor.py:153} INFO - Started process (PID=2277) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:32:02.757+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:32:02.758+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:32:02.757+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:32:02.777+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:32:02.776+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:32:02.777+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:32:02.777+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:32:02.777+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:32:02.786+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-04T22:32:40.231+0100] {processor.py:153} INFO - Started process (PID=2316) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:32:40.232+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:32:40.232+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:32:40.232+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:32:40.246+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:32:40.245+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:32:40.246+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:32:40.246+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:32:40.246+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:32:40.254+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:33:15.449+0100] {processor.py:153} INFO - Started process (PID=2352) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:33:15.450+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:33:15.450+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:33:15.450+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:33:15.464+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:33:15.463+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:33:15.464+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:33:15.464+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:33:15.464+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:33:15.472+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:33:46.529+0100] {processor.py:153} INFO - Started process (PID=2382) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:33:46.530+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:33:46.531+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:33:46.531+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:33:46.547+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:33:46.547+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:33:46.547+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:33:46.547+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:33:46.547+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:33:46.555+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-01-04T22:34:24.191+0100] {processor.py:153} INFO - Started process (PID=2421) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:34:24.192+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:34:24.193+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:34:24.193+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:34:24.206+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:34:24.206+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:34:24.206+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:34:24.206+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:34:24.207+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:34:24.214+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:34:59.261+0100] {processor.py:153} INFO - Started process (PID=2455) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:34:59.263+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:34:59.264+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:34:59.264+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:34:59.283+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:34:59.282+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:34:59.283+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:34:59.283+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:34:59.283+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:34:59.291+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-04T22:35:30.071+0100] {processor.py:153} INFO - Started process (PID=2484) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:35:30.072+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:35:30.072+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:35:30.072+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:35:30.086+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:35:30.085+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:35:30.086+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:35:30.086+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:35:30.086+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:35:30.094+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:36:07.214+0100] {processor.py:153} INFO - Started process (PID=2521) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:36:07.215+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:36:07.216+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:36:07.216+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:36:07.231+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:36:07.231+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:36:07.232+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:36:07.231+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:36:07.232+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:36:07.239+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:36:43.657+0100] {processor.py:153} INFO - Started process (PID=2556) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:36:43.658+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:36:43.659+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:36:43.658+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:36:43.672+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:36:43.672+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:36:43.672+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:36:43.672+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:36:43.672+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:36:43.680+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-01-04T22:37:17.136+0100] {processor.py:153} INFO - Started process (PID=2588) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:37:17.137+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:37:17.138+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:37:17.137+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:37:17.160+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:37:17.160+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:37:17.161+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:37:17.161+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:37:17.161+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:37:17.170+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-01-04T22:37:56.518+0100] {processor.py:153} INFO - Started process (PID=2626) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:37:56.519+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:37:56.519+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:37:56.519+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:37:56.533+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:37:56.533+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:37:56.533+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:37:56.533+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:37:56.534+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:37:56.542+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:38:36.158+0100] {processor.py:153} INFO - Started process (PID=2663) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:38:36.159+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:38:36.160+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:38:36.160+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:38:36.173+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:38:36.173+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:38:36.174+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:38:36.174+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:38:36.174+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:38:36.182+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:39:13.591+0100] {processor.py:153} INFO - Started process (PID=2692) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:39:13.592+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:39:13.593+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:39:13.593+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:39:13.610+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:39:13.609+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:39:13.610+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:39:13.610+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:39:13.610+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:39:13.619+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-04T22:39:45.182+0100] {processor.py:153} INFO - Started process (PID=2723) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:39:45.184+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:39:45.185+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:39:45.184+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:39:45.203+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:39:45.202+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:39:45.203+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:39:45.203+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:39:45.203+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:39:45.212+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-04T22:40:23.161+0100] {processor.py:153} INFO - Started process (PID=2760) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:40:23.162+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:40:23.162+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:40:23.162+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:40:23.176+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:40:23.176+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:40:23.177+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:40:23.177+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:40:23.177+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:40:23.185+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:41:02.005+0100] {processor.py:153} INFO - Started process (PID=2797) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:41:02.007+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:41:02.008+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:41:02.007+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:41:02.023+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:41:02.023+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:41:02.023+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:41:02.023+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:41:02.023+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:41:02.032+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:41:39.905+0100] {processor.py:153} INFO - Started process (PID=2834) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:41:39.907+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:41:39.907+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:41:39.907+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:41:39.923+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:41:39.923+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:41:39.923+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:41:39.923+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:41:39.923+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:41:39.931+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:42:14.746+0100] {processor.py:153} INFO - Started process (PID=2867) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:42:14.748+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:42:14.749+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:42:14.749+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:42:14.768+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:42:14.767+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:42:14.768+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:42:14.768+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:42:14.768+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:42:14.776+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-01-04T22:42:51.408+0100] {processor.py:153} INFO - Started process (PID=2903) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:42:51.409+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:42:51.410+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:42:51.410+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:42:51.427+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:42:51.427+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:42:51.427+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:42:51.427+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:42:51.428+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:42:51.436+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-04T22:43:30.439+0100] {processor.py:153} INFO - Started process (PID=2941) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:43:30.440+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:43:30.441+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:43:30.441+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:43:30.459+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:43:30.459+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:43:30.460+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:43:30.459+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:43:30.460+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:43:30.468+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-04T22:44:09.517+0100] {processor.py:153} INFO - Started process (PID=2978) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:44:09.519+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:44:09.519+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:44:09.519+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:44:09.539+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:44:09.539+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:44:09.539+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:44:09.539+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:44:09.540+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:44:09.548+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-01-04T22:44:49.473+0100] {processor.py:153} INFO - Started process (PID=3016) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:44:49.474+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:44:49.475+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:44:49.475+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:44:49.490+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:44:49.489+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:44:49.490+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:44:49.490+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:44:49.490+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:44:49.498+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:45:20.980+0100] {processor.py:153} INFO - Started process (PID=3046) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:45:20.980+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:45:20.981+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:45:20.981+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:45:20.995+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:45:20.995+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:45:20.995+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:45:20.995+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:45:20.995+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:45:21.004+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:45:56.912+0100] {processor.py:153} INFO - Started process (PID=3082) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:45:56.913+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:45:56.914+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:45:56.914+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:45:56.928+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:45:56.928+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:45:56.928+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:45:56.928+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:45:56.928+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:45:56.937+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:46:29.677+0100] {processor.py:153} INFO - Started process (PID=3115) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:46:29.678+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:46:29.679+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:46:29.678+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:46:29.693+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:46:29.693+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:46:29.693+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:46:29.693+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:46:29.693+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:46:29.702+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:47:04.318+0100] {processor.py:153} INFO - Started process (PID=3149) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:47:04.319+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:47:04.320+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:47:04.320+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:47:04.335+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:47:04.334+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:47:04.335+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:47:04.335+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:47:04.335+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:47:04.343+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:47:39.589+0100] {processor.py:153} INFO - Started process (PID=3184) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:47:39.590+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:47:39.591+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:47:39.590+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:47:39.607+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:47:39.606+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:47:39.607+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:47:39.607+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:47:39.607+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:47:39.622+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-01-04T22:48:15.128+0100] {processor.py:153} INFO - Started process (PID=3219) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:48:15.129+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:48:15.130+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:48:15.130+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:48:15.146+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:48:15.145+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:48:15.146+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:48:15.146+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:48:15.146+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:48:15.154+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:48:55.500+0100] {processor.py:153} INFO - Started process (PID=3257) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:48:55.501+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:48:55.502+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:48:55.501+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:48:55.517+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:48:55.517+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:48:55.517+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:48:55.517+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:48:55.517+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:48:55.525+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-01-04T22:49:33.653+0100] {processor.py:153} INFO - Started process (PID=3294) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:49:33.654+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:49:33.655+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:49:33.655+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:49:33.669+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:49:33.669+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:49:33.670+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:49:33.669+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:49:33.670+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:49:33.678+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-01-04T22:50:08.915+0100] {processor.py:153} INFO - Started process (PID=3327) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:50:08.917+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:50:08.918+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:50:08.917+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:50:08.938+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:50:08.938+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:50:08.939+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:50:08.938+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:50:08.939+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:50:08.948+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-01-04T22:50:39.428+0100] {processor.py:153} INFO - Started process (PID=3356) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:50:39.428+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:50:39.429+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:50:39.429+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:50:39.443+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:50:39.442+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:50:39.443+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:50:39.443+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:50:39.443+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:50:39.451+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-01-04T22:51:11.042+0100] {processor.py:153} INFO - Started process (PID=3385) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:51:11.043+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:51:11.044+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:51:11.044+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:51:11.059+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:51:11.059+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:51:11.059+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:51:11.059+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:51:11.060+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:51:11.068+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-01-04T22:51:42.806+0100] {processor.py:153} INFO - Started process (PID=3416) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:51:42.807+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:51:42.809+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:51:42.808+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:51:42.826+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:51:42.825+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:51:42.826+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:51:42.826+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:51:42.826+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:51:42.834+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-01-04T22:52:21.551+0100] {processor.py:153} INFO - Started process (PID=3454) to work on /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:52:21.552+0100] {processor.py:743} INFO - Processing file /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-01-04T22:52:21.552+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:52:21.552+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:52:21.566+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:52:21.566+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-01-04T22:52:21.566+0100] {logging_mixin.py:137} INFO - [2023-01-04T22:52:21.566+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-01-04T22:52:21.567+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-01-04T22:52:21.574+0100] {processor.py:175} INFO - Processing /home/chrisfleig/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
